typescript/src/feast/pubsub/google.ts

```diff
@@ -0,0 +1,7 @@
+import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda";
```

Comment:
Very minor, but is it worth being explicit that these are types?

```suggestion
import type {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda";
```

Reply:
Good point